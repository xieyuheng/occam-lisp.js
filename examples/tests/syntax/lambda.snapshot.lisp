(lambda (x) x)

(define apply (lambda (f x) (f x)))
apply
(apply (lambda (x) x))
(apply (lambda (x) x) (lambda (x) x))
