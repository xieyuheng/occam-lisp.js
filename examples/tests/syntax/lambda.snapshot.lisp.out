(lambda (x) x)
(lambda (f x) (f x))
((lambda (f x) (f x)) (lambda (x) x))
(lambda (x) x)
